turbo.json

```diff
@@ -36,6 +36,10 @@
             "inputs": ["rollup.config.types.js", "tsconfig.*", "src/**"],
             "outputs": ["declarations/**", "dist/**/*.d.ts", "lib/**/*.d.ts"]
         },
+        "lint:fix": {
+            "inputs": ["src/**", "test/**"],
```

Comment:
Should we be passing `test/*` to the `eslint --fix` and `prettier -w` calls too?

Reply:
there arent any `test/` dirs tho, other than `library-legacy` (which i didnt add this command for because it has its own flows and is a separate library). everything is in `src/`

Reply:
i removed `test/**` from this bc id just naively copy-pasted it